Dockerfile

```diff
@@ -36,7 +36,8 @@ CMD cd /mnt/ctlog/ && \
         --trusted_cert_file=/usr/local/etc/ctlog_ca_roots.pem \
         --stderrthreshold=0 \
         --log_dir=/mnt/ctlog/logs \
-        --tree_signing_frequency_seconds=60 \
+        --tree_signing_frequency_seconds=30 \
+        --guard_window_seconds=10 \
```

Comment:
That's for testing only, no? Sure you want to push this, especially in this PR (that looks like this hasn't got much to do with it)?